### DISCUSSION TITLE
Component Thumbnails

### DISCUSSION THREAD

**Discussion by karlbaumhauer** [flyntwp/flynt#355] [Q&A] [answered]
Hi,

first of all I want to let you know how fascinated I am about the flynt project. I started to work on a small website project and only was used to build frontend components before. It made it awesomely easy for me to start here. 

Before I open an issue I would like to ask if there are any specifications for the screenshot images as component thumbnails. For some reason 2 of my thumbnails dont show up:
![Bildschirmfoto 2021-01-17 um 20 33 54](https://user-images.githubusercontent.com/48211526/104853788-61789b00-5903-11eb-8bf1-c76d4847393f.png)
![Bildschirmfoto 2021-01-17 um 20 33 47](https://user-images.githubusercontent.com/48211526/104853791-62a9c800-5903-11eb-9354-4bea030d77d0.png)

Any idea why? I am pretty sure they where showing up about a week ago...;-)

thx and cheers,
Karl

**Comment by steffenbew** [comment, marked as answer]
Hi Karl,

I'm very happy to hear that getting started with Flynt was easy for you!

Here's what I can say about potential issues with preview thumbnails:

- Screenshots need to be named `screenshot.png` (all lowercase).
- Screenshots _have_ to be PNGs, other types (like JPGs) don't work. The filename [is hardcoded](https://github.com/flyntwp/flynt/blob/master/Components/FeatureAdminComponentScreenshots/functions.php#L30) in the [feature](https://github.com/flyntwp/flynt/tree/master/Components/FeatureAdminComponentScreenshots).
- Screenshot names must have URL friendly characters only (no umlauts or similar).

If it's none of the above, there might be a caching issue, or a bug with the [feature](https://github.com/flyntwp/flynt/tree/master/Components/FeatureAdminComponentScreenshots).

Any error messages when opening the browser's dev tools / inspector?

Best,
Steffen

**Reply by karlbaumhauer** [reply]
Hi Steffen, 

no idea why and I did not change anything but it worked yesterday. 
I will keep that in mind and try to regularly take a look on it. I suppose it to be a caching issue (even though I disabled my cache).

Nevertheless, thanks for your advise 
Karl
